spec/models/grant_submission_spec.rb

```diff
@@ -61,10 +61,10 @@
 
   # TODO: need to add a test for comparing csv of requests with csv levels
   context 'validations' do
-    context 'with requested_funding_dollars greater than grant limit' do
+    context 'with funding_requests_csv greater than grant limit' do
       it 'is not valid' do
         expect(subject).to be_valid
-        subject.requested_funding_dollars = subject.grant.max_funding_dollars + 1
+        subject.funding_requests_csv = subject.grant.funding_levels_csv.to_i + 1
```

Comment:
this is a more useful test, but again confirming that the funding_requests_csv is valid is much more tricky because you have to parse the string.  See the TODO note at line 62 :)